Description: Fix spelling errors.
 * Agressive  -> Aggressive
 * deamon     -> daemon
 * occuring   -> occurring
 * trucating  -> truncating
 * preceeding -> preceding
 * occured    -> occurred
 * occure     -> occur
 * Unkown     -> Unknown
 * triggerd   -> triggered
Author: Bas Couwenberg <sebastic@debian.org>
Forwarded: https://github.com/Icinga/icinga-core/pull/10
Applied-Upstream: https://github.com/Icinga/icinga-core/commit/0b216eaa5ca072f4c7c3b29a76f417c1593f9038

--- a/html/docs/en/tuning.html
+++ b/html/docs/en/tuning.html
@@ -146,8 +146,8 @@
         need to schedule regular checks of your hosts (see above). More information on cached checks can be found <a class="link" href="cachedchecks.html" title="7.21. Cached Checks">here</a>.</p>
       </li>
 <li class="listitem">
-        <p><span class="bold"><strong>Don't use agressive host checking</strong></span>. Unless you're having problems with Icinga
-        recognizing host recoveries, We would recommend not enabling the <a class="link" href="configmain.html#configmain-use_agressive_host_checking">use_aggressive_host_checking</a> option. With this option turned off host checks
+        <p><span class="bold"><strong>Don't use aggressive host checking</strong></span>. Unless you're having problems with Icinga
+        recognizing host recoveries, We would recommend not enabling the <a class="link" href="configmain.html#configmain-use_aggressive_host_checking">use_aggressive_host_checking</a> option. With this option turned off host checks
         will execute much faster, resulting in speedier processing of service check results. However, host recoveries can be missed under
         certain circumstances when this it turned off. For example, if a host recovers and all of the services associated with that host
         stay in non-OK states (and don't "wobble" between different non-OK states), Icinga may miss the fact that the host has
--- a/html/docs/de/tuning.html
+++ b/html/docs/de/tuning.html
@@ -150,7 +150,7 @@
       </li>
 <li class="listitem">
         <p><span class="bold"><strong>Nutzen Sie keine agressiven Host-Prüfungen</strong></span>. Solange Sie keine Probleme damit haben, dass
-        Icinga Host-Erholungen nicht korrekt erkennt, würden wir empfehlen, die <a class="link" href="configmain.html#configmain-use_agressive_host_checking">use_aggressive_host_checking</a>-Option nicht zu aktivieren. Wenn diese Option
+        Icinga Host-Erholungen nicht korrekt erkennt, würden wir empfehlen, die <a class="link" href="configmain.html#configmain-use_aggressive_host_checking">use_aggressive_host_checking</a>-Option nicht zu aktivieren. Wenn diese Option
         abgeschaltet ist, werden Host-Prüfungen viel schneller ausgeführt, was zu schnellerer Ausführung von Service-Prüfungen führt.
         Allerdings können Host-Erholungen unter bestimmten Umständen übersehen werden, wenn sie ausgeschaltet ist. Wenn sich z.B. der Host
         erholt, aber alle mit ihm verbundenen Services in einem nicht-OK-Zustand bleiben (und nicht zwischen verschiedenen
--- a/html/docs/en/configmain.html
+++ b/html/docs/en/configmain.html
@@ -1434,7 +1434,7 @@
 <tr><td align="left" valign="top">
     <p>This is an advanced feature. This option determines how many buffer slots Icinga will reserve for caching external commands
     that have been read from the external command file by a worker thread, but have not yet been processed by the main thread of the
-    Icinga deamon. Each slot can hold one external command, so this option essentially determines how many commands can be buffered.
+    Icinga daemon. Each slot can hold one external command, so this option essentially determines how many commands can be buffered.
     For installations where you process a large number of passive checks (e.g. <a class="link" href="distributed.html" title="7.6. Distributed Monitoring">distributed setups</a>), you
     may need to increase this number. You should consider using PNP4Nagios to graph Icinga's usage of external command buffers. You
     can read more on how to configure graphing <a class="link" href="perfgraphs.html" title="8.7. Graphing Performance Info With PNP4Nagios">here</a>.</p>
--- a/html/docs/de/icinga-web-api.html
+++ b/html/docs/de/icinga-web-api.html
@@ -290,7 +290,7 @@ order(SERVICE_CURRENT_STATE;DESC)/countC
        &lt;column name="SERVICE_INSTANCE_ID"&gt;1&lt;/column&gt;
        &lt;column name="SERVICE_NAME"&gt;MailQ&lt;/column&gt;
        &lt;column name="SERVICE_DISPLAY_NAME"&gt;MailQ&lt;/column&gt;
-       &lt;column name="SERVICE_OUTPUT"&gt;Error occured:error=1:0:0&lt;/column&gt;
+       &lt;column name="SERVICE_OUTPUT"&gt;Error occurred:error=1:0:0&lt;/column&gt;
        &lt;column name="SERVICE_PERFDATA"&gt;&lt;/column&gt;
    &lt;/result&gt;
    &lt;result&gt;
@@ -321,7 +321,7 @@ order(SERVICE_CURRENT_STATE;DESC)/countC
        "SERVICE_INSTANCE_ID":"1",
        "SERVICE_NAME":"MailQ",
        "SERVICE_DISPLAY_NAME":"MailQ",
-       "SERVICE_OUTPUT":"Error occured:error=1:0:0",
+       "SERVICE_OUTPUT":"Error occurred:error=1:0:0",
        "SERVICE_PERFDATA":"" 
    },{
        "SERVICE_ID":"311",
--- a/html/docs/de/sample-cgi.html
+++ b/html/docs/de/sample-cgi.html
@@ -611,7 +611,7 @@ highlight_table_rows=1
 # If you want to log information from cgi's (e.g. all submitted commands)
 # then set this option to 1, default is 0 (off).
 # WARNING:
-# This log is highly experimental and changes may occure without notice. Use at your own risk!!
+# This log is highly experimental and changes may occur without notice. Use at your own risk!!
 
 use_logging=@ENABLECGILOG@
 
--- a/html/docs/en/cachedchecks.html
+++ b/html/docs/en/cachedchecks.html
@@ -102,7 +102,7 @@
 
     <p>When Icinga needs to perform an on-demand host or service check, it will make a determination as to whether it can used a
     cached check result or if it needs to perform an actual check by executing a plugin. It does this by checking to see if the last check
-    of the host or service occured within the last X minutes, where X is the cached host or service check horizon.</p>
+    of the host or service occurred within the last X minutes, where X is the cached host or service check horizon.</p>
 
     <p>If the last check was performed within the timeframe specified by the cached check horizon variable, Icinga will use the
     result of the last host or service check and will <span class="emphasis"><em>not</em></span> execute a new check. If the host or service has not yet been
--- a/html/docs/en/icinga-web-api.html
+++ b/html/docs/en/icinga-web-api.html
@@ -283,7 +283,7 @@ order(SERVICE_CURRENT_STATE;DESC)/countC
        &lt;column name="SERVICE_INSTANCE_ID"&gt;1&lt;/column&gt;
        &lt;column name="SERVICE_NAME"&gt;MailQ&lt;/column&gt;
        &lt;column name="SERVICE_DISPLAY_NAME"&gt;MailQ&lt;/column&gt;
-       &lt;column name="SERVICE_OUTPUT"&gt;Error occured:error=1:0:0&lt;/column&gt;
+       &lt;column name="SERVICE_OUTPUT"&gt;Error occurred:error=1:0:0&lt;/column&gt;
        &lt;column name="SERVICE_PERFDATA"&gt;&lt;/column&gt;
    &lt;/result&gt;
    &lt;result&gt;
@@ -314,7 +314,7 @@ order(SERVICE_CURRENT_STATE;DESC)/countC
        "SERVICE_INSTANCE_ID":"1",
        "SERVICE_NAME":"MailQ",
        "SERVICE_DISPLAY_NAME":"MailQ",
-       "SERVICE_OUTPUT":"Error occured:error=1:0:0",
+       "SERVICE_OUTPUT":"Error occurred:error=1:0:0",
        "SERVICE_PERFDATA":"" 
    },{
        "SERVICE_ID":"311",
--- a/html/docs/en/sample-cgi.html
+++ b/html/docs/en/sample-cgi.html
@@ -611,7 +611,7 @@ highlight_table_rows=1
 # If you want to log information from cgi's (e.g. all submitted commands)
 # then set this option to 1, default is 0 (off).
 # WARNING:
-# This log is highly experimental and changes may occure without notice. Use at your own risk!!
+# This log is highly experimental and changes may occur without notice. Use at your own risk!!
 
 use_logging=@ENABLECGILOG@
 
--- a/cgi/cmd.c
+++ b/cgi/cmd.c
@@ -1816,7 +1816,7 @@ void request_command_data(int cmd) {
 		print_form_element(PRINT_AUTHOR, cmd);
 		print_form_element(PRINT_COMMENT_BOX, cmd);
 
-		snprintf(help_text, sizeof(help_text), "Define here if this downtime should get triggerd by another downtime of a particular %s.", (cmd == CMD_PROCESS_HOST_CHECK_RESULT) ? "host" : "service");
+		snprintf(help_text, sizeof(help_text), "Define here if this downtime should get triggered by another downtime of a particular %s.", (cmd == CMD_PROCESS_HOST_CHECK_RESULT) ? "host" : "service");
 		help_text[sizeof(help_text)-1] = '\x0';
 
 		printf("<tr id=\"trigger_select\"><td class=\"objectDescription descriptionleft\">Triggered By:");
@@ -1854,7 +1854,7 @@ void request_command_data(int cmd) {
 		printf("\t</select>\n");
 		printf("</td></tr>\n");
 
-		/* hide "Triggerd by" selction if nothing is found to get triggerd from */
+		/* hide "Triggerd by" selction if nothing is found to get triggered from */
 		if (!found_trigger_objects)
 			printf("<tr style=\"display:none;\"><td colspan='2'><script language=\"JavaScript\">document.getElementById('trigger_select').style.display = 'none';</script></td></tr>\n");
 
